interfaces/console/src/app/console/sites/[id]/page.tsx

```diff
@@ -89,16 +77,21 @@ interface SiteDetailsProps {
 
 const Page: React.FC<SiteDetailsProps> = ({ params }) => {
   const { id } = params;
-  const [site, setSite] = useState<TSiteForm>(SITE_INIT);
+  const router = useRouter();
+  const subscriptionsRef = useRef<Record<string, boolean>>({});
+
   const [activeSite, setActiveSite] = useState<SiteDto>(defaultSite);
-  const [openSiteConfig, setOpenSiteConfig] = useState(false);
-  const [componentsList, setComponentsList] = useState<any[]>([]);
   const [selectedSiteId, setSelectedSiteId] = useState<string | null>(null);
```

Comment:
What's the difference between `activeSite` and `selectedSiteId`, Isn't `activeSite.Id` is the same as `selectedSiteId`

Reply:
We now have only two states: activeViewComp and activeSite.